lib/utils/disposable.dart

```diff
@@ -79,12 +79,12 @@ class StreamSubscriptions implements Disposable {
     for (StreamSubscription subscription in _subscriptions) {
       if (catchExceptions) {
         try {
-          subscription.cancel();
+          subscription?.cancel();
```

Comment:
I assume this means that some client is passing in `null` to the `add()` method above?

I wonder if we should not add null subscriptions to the `_subscriptions` list, and or log when we receive a null arg?

Reply:
Good idea.  Done.